### DISCUSSION TITLE
During the fine-tuning of the model, the RMSE failed to reach the target, yet overfitting had already occurred.

### DISCUSSION THREAD

**Discussion by ContiMdc** [ACEsuit/mace#1430] [Q&A]
<img width="884" height="584" alt="图片1" src="https://github.com/user-attachments/assets/7ac057dc-e3bc-4b3d-902b-60983ea6be89" />

Hi everyone
    I am using a small training set consisting of 1,000 structures to fine-tune the MH-one model. However, the model converged after just 51 steps, and subsequent overfitting was observed. The RMSE for the forces did not meet the required criteria. Has anyone else encountered this situation? How should I resolve it?
    The following are the parameters and log files for my fine-tuned model.

_**Parameters of Multihead finetuning (MH-one)**_

    "--name", "MACE",
    "--foundation_model", "mace-mh-1.model",
    "--foundation_head", "omat_pbe",
    "--E0s", "{1:-1.1173640, 6:-1.2615544, 8:-1.5475214}",

    "--pt_train_file", "npg_train_combined.extxyz",
    "--train_file", "npg_train.extxyz",
    "--valid_fraction", "0.05",
    "--test_file", "npg_test.extxyz",

    "--multiheads_finetuning", "True",

    "--atomic_numbers", "[1,6,8]",

    "--energy_weight", "1.0",
    "--forces_weight", "100.0",

    "--scaling", "rms_forces_scaling",
    "--batch_size", "2",
    "--max_num_epochs", "300",

    "--swa",
    "--start_swa", "200",
    "--swa_energy_weight", "10.0",
    "--swa_forces_weight", "100.0",

    "--ema",
    "--ema_decay", "0.99999",
    "--amsgrad",

    "--default_dtype", "float64",
    "--device", "cuda",
    "--seed", "123",

    "--compute_stress", "True",
    "--stress_key", "REF_stress"
[1815.log](https://github.com/user-attachments/files/26433152/1815.log)

**Comment by ilyes319** [comment]
Hey @ContiMdc, looking at your log, things seem fine actually. What do you mean by "forces did not meet the required criteria"?

**Reply by ContiMdc** [reply]
I would like to apologize for the delayed response. The issue I previously reported has now been resolved. After 52 epochs, the energy error began to increase, while the force error had not yet reached the target threshold of 20 meV/Å. This suggests that the current training configuration does not yield sufficiently satisfactory performance.
To mitigate this issue, I reduced the sizes of both the replay and fine-tuning datasets. Furthermore, given the specific characteristics of the dataset, the application of SWA resulted in increased errors; therefore, it was excluded from the training procedure. The model exhibited overfitting before the energy error could reach the target accuracy of 1 meV/atom, which can be attributed to the presence of several energy outliers in the dataset.
